custom_metrics/css.js

```diff
@@ -0,0 +1,24 @@
+//[css]
+// Uncomment the previous line for testing on webpagetest.org
+
+return JSON.stringify({
+  'has_cssinjs_styled_components': !!(document.querySelector('style[data-styled]') || document.querySelector('style[data-styled-components]')),
```

Comment:
Weird, one of my comments got lost.

I wanted to ask if you could combine all of these into a single array property rather than having one property per tool. Example output:

```json
{
  "css_in_js": ["Goober", "JSS"]
}
```

This makes it easier to tell if there are _any_ CSS-in-JS tools in use by checking the length of the array rather than testing each individual property.

Reply:
Yeah, was thinking about. How all this will be represented in BigQuery? Would it be still easy to calculate popularity of each framework?

Reply:
Yes, the query could easily sort by the most popular tool names in the array. I can show you some examples when we get there.

Reply:
Please, have a look at new solution. As discussed it returns an array with frameworks used. A quick one, do you think it would be better to return empty array or null if there is no libraries used?

Reply:
Empty array sounds good

Reply:
It's empty array now, so resolving this one :)